packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-summary-section/order-summary-section.tsx

```diff
@@ -96,45 +100,61 @@ export const OrderSummarySection = ({ order }: OrderSummarySectionProps) => {
     return false
   }, [reservations])
 
+  // TODO: We need a way to link payment collections to a change order to
```

Comment:
thought: Could the link have an additional column `type` or something like that?

Reply:
yes, we can do that